src/FsLibLog/FsLibLog.fs

```diff
@@ -789,3 +789,54 @@ module LogProvider =
         let stackFrame = StackFrame(2, false)
         getLoggerByType(stackFrame.GetMethod().DeclaringType)
 
+/// Provides operators to make writing logs more streamlined.
+module Operators =
+
+    /// <summary>
+    /// Amend a log with a message. Wrapper for `Log.setMessage`.
+    /// </summary>
+    /// <param name="message">The `string` of the base message.</param>
+    /// <returns>A new Log instance with the specified message.</returns>
+    let (!!) message = Log.setMessage message
+
+    /// <summary>
+    /// Amends a log with a parameter. Wrapper for `Log.addParameter`.
+    /// </summary>
+    /// <param name="log">The Log to add the parameter to.</param>
+    /// <param name="value">The value for the parameter.</param>
+    /// <returns>The Log with the added parameter.</returns>
+    let (>>!) log value = log >> Log.addParameter value
+
+    /// <summary>
+    /// Amends a Log with additional named parameters for context. This helper adds more context to a log.
+    /// This DOES NOT affect the parameters set for a message template. This is the same calling OpenMappedContext right before logging.
+    ///
+    /// Wrapper for `Log.addContext`.
+    /// </summary>
+    /// <param name="log">The log to add the parameter to.</param>
+    /// <param name="key">The name for the parameter.</param>
+    /// <param name="value">The value for the parameter.</param>
+    /// <returns>The amended log with the parameter added.</returns>
+    let (>>!-) log (key, value) = log >> Log.addContext key value
+
+    /// <summary>
+    /// Amends a Log with additional named parameters for context. This helper adds more context to a log. This DOES NOT affect the parameters set for a message template.
+    /// This is the same calling OpenMappedContext right before logging. This destructures an object rather than calling ToString() on it. WARNING: Destructring can be expensive.
+    ///
+    /// Wrapper for `Log.addContextDestructured`.
```

Comment:
I think you'll want to use the `<see>` tag to get the link setup correctly https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/language-specification/documentation-comments#see

Reply:
Neither Ionide (mostly working) nor Rider (no XML handling in F#, apparently) provided a suitable testing ground to ensure these work 100%, but following [the docs](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/), I _think_ I got it right?